go/private/rules/transition.bzl

```
@@ -54,6 +64,14 @@ def filter_transition_label(label):
     else:
         return str(Label(label))
 
+def _original_value_setting(setting):
```

```suggestion
def _original_setting_key(setting):
```
These are keys to store original settings, right?